primitives/runtime/src/generic/digest.rs

```diff
@@ -184,6 +192,8 @@ pub enum DigestItemRef<'a, Hash: 'a> {
 	ChangesTrieSignal(&'a ChangesTrieSignal),
 	/// Any 'non-system' digest item, opaque to the native code.
 	Other(&'a Vec<u8>),
+	/// Runtime code or heap pages updated.
+	RuntimeCodeOrHeapPagesUpdated,
```

Comment:
Right now the story about heap pages is in flux. AFAIU, there is an interest to get rid of heap pages and or change the semantics at least.

In that case, maybe it would be better to split those digests in two: runtime code updated and heap pages updated? Then, if needed we deprecate the heap pages. The alternative would be to leave it as is and then change the semantics of `RuntimeCodeOrHeapPagesUpdated` which seems a bit messier.

Reply:
I preferred `RuntimeUpdated`, as it simply means "you might want to check whatever storage item concerns the runtime". The ambiguity is advantageous here.

Reply:
@thiolliere @bkchr are you guys okay to revert back to `RuntimeUpdated` so that we can be a bit more flexible regarding what that actually means? My preference would be to have a single enum variant, especially if the heap pages one might get deprecated in the future.

Reply:
I also think that once we remove HeapPages stuff, RuntimeCodeOrHeapPagesUpdated always mean RuntimeCodeUpdated.
So the transition shouldn't be that messy. Anyway it is ok to me.

Reply:
Reverted back to `RuntimeUpdated`.